SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -35,7 +36,45 @@ DECLARE_CYCLE_STAT(TEXT("Projection"), STAT_Projection, STATGROUP_SpatialDebugge
 DECLARE_CYCLE_STAT(TEXT("DrawIcons"), STAT_DrawIcons, STATGROUP_SpatialDebugger);
 DECLARE_CYCLE_STAT(TEXT("DrawText"), STAT_DrawText, STATGROUP_SpatialDebugger);
 DECLARE_CYCLE_STAT(TEXT("BuildText"), STAT_BuildText, STATGROUP_SpatialDebugger);
-DECLARE_CYCLE_STAT(TEXT("SortingActors"), STAT_SortingActors, STATGROUP_SpatialDebugger);
+
+namespace SpatialGDK
+{
+class FSubView;
+struct SpatialDebugging;
+} // namespace SpatialGDK
+
+class FSpatialDebuggerSystem
```

Comment:
1. `FSpatialDebuggerSystem` is in the same vein as `ActorSystem` but sounds funny to me, especially just next to `ASpatialDebugger`.
1. Other Spatial-interacting classes are in `SpatialGDK` namespace and don't follow Unreal's `F`-prefixed naming for non-UCLASSes. Should I go for this naming here as well?

Reply:
1. I agree it's a bit of a mouthful, but fairly explanatory, so I'm fine with it
2. Yep, no need to append the F/U/A etc. except when following Unreal's coding convention to detail UClass/UStruct and class hierarchy.

Reply:
2. Yeah, this is what I've been getting at - https://docs.unrealengine.com/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/#namingconventions details 
> Type names are prefixed with an additional upper-case letter
and
> Most other classes are prefixed by F
They don't really use classes without prefixes AFAIK.

Reply:
It does seem to be a convention in the GDK code though - should I move it to `SpatialGDK` and rename `SpatialDebuggerSystem`?